README.md

```diff
@@ -23,8 +23,10 @@ Usage
 -----
 
 - `:Translate en:fr`
-    - translate **english → french** (see [here](https://github.com/soimort/translate-shell#code-list) for code list)
+    - translate **english → french** (see [here](https://github.com/soimort/translate-shell#code-list) for code list) and put translation in a buffer
     
+- `:'<,'>TranslateReplace en:fr`
+    - translate **english → french** (see [here](https://github.com/soimort/translate-shell#code-list) for code list) and replace the selection
```

Comment:
Can you remove the link to code list ? 😛 it's written just above

Reply:
Done !